packages/picasso.js/src/core/component/component-factory.js

```diff
@@ -363,6 +363,11 @@ function componentFactory(definition, context = {}) {
     rend.clear();
   };
 
+  fn.setVisibility = (isVisible) => {
+    instanceContext.visible = () => isVisible;
```

Comment:
expose `visible` via the prepareContext function and add them the keyword to the reserved prop list.